### DISCUSSION TITLE
Uso de format en lista de elementos

### DISCUSSION THREAD

**Discussion by belusz** [santiemanuel/milProgramadoresQA#2] [Preguntas y Respuestas] [answered]
Hola, acá está mi resolución del ejercicio 2 de los extra de f-strings, quería saber si está bien la sintaxis usada para mostrar una lista de números.

```python
coma=','
cad="1 {} 2 {} 3 {} 4"
cad=cad.format(coma,coma,coma)
print(cad)
```

**Comment by santiemanuel** [comment, marked as answer]
Buenas, Belén, en este ejercicio la idea es mostrar una lista de valores diferentes insertados por la función `format` en las cadenas.

En este caso lo que tendrías que hacer es pasar los valores separados por coma dentro de los paréntesis, y la cadena inicial contenga las comas que se van a visualizar.
De este modo:
```python
cadena = "{}, {}, {}, {}"
cadena = cadena.format(1, 2, 3, 4)
print(cadena)
```

Que en este caso nos da la salida:
```
1, 2, 3, 4
```
